### PR TITLE
feat: [OIP-408]: Grafana Loki initial config

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/services/cv/index.tsx [hunk 1]
```diff
@@ -5927,7 +5938,7 @@ export interface SLIOnboardingGraphs {
 }
 
 export interface SLOConsumptionBreakdown {
-  contributedErrorBudgetBurned: number
+  contributedErrorBudgetBurned?: number
```
Comment (praneshg239) [inline src/services/cv/index.tsx:5941]: @arpit-harness @Varsha-lalwani Did we make any change in backend that causes this change in SLOs? It is breaking the frontend type check.

Comment (praneshg239) [reply src/services/cv/index.tsx:5941]: fixed

File: src/modules/85-cv/pages/slos/CVSLODetailsPage/DetailsPanel/views/CompositeSLOConsumption/CompositeSLOConsumption.utils.tsx [hunk 2]
```diff
@@ -124,7 +124,7 @@ export const RenderActualSlo: Renderer<CellProps<any>> = ({ row }) => {
   )
 }
 
-export const durationAsString = (consumptionMinutes: number): string => {
+export const durationAsString = (consumptionMinutes?: number): string => {
```
Comment (harshilbits) [inline src/modules/85-cv/pages/slos/CVSLODetailsPage/DetailsPanel/views/CompositeSLOConsumption/CompositeSLOConsumption.utils.tsx:127]: This change is out of scope of current PR.

Comment (praneshg239) [reply src/modules/85-cv/pages/slos/CVSLODetailsPage/DetailsPanel/views/CompositeSLOConsumption/CompositeSLOConsumption.utils.tsx:127]: but this change is needed as it affects the swagger

**Review by deepesh-ui — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]
